srcpkgs/eternity/INSTALL.msg

```diff
@@ -0,0 +1,15 @@
+Before loading WADs, go to:
+
+	/usr/share/eternity/base
+
+And check if the following directories exist:
+
+	* doom
+	* doom2
+	* hacx
+	* heretic
+	* plutonia
+	* shots
+	* tnt
```

Comment:
`make_dirs` should be used in the template instead, though the software needing these directories to exist doesn't fill me with confidence.

Reply:
If one clones the repository those directories are there. This problem happens with the release tar.